model/qti/ImportService.php

```diff
@@ -328,7 +328,7 @@ public function importQTIPACKFile($file, core_kernel_classes_Class $itemClass, $
                         }
                         
                         // replace uri if changed
-                        if ($auxPath != $info['uri']) {
+                        if ($auxPath != ltrim($info['uri'], '/')) {
```

Comment:
After importing interaction all paths starting with PCI interaction identifier (as described [here](http://forge.taotesting.com/projects/tao/wiki/TAO_PCI_and_PIC#Implementing-pciCreatorjs) we can use relative paths staring with a PCI typeIdentifier) will be prefixed with `/`.
for example:
`<pci:portableCustomInteraction customInteractionTypeIdentifier="adaptiveChoiceInteraction" hook="adaptiveChoiceInteraction/runtime/adaptiveChoiceInteraction.js">`
will be replaced to 
`<pci:portableCustomInteraction customInteractionTypeIdentifier="adaptiveChoiceInteraction" hook="/adaptiveChoiceInteraction/runtime/adaptiveChoiceInteraction.js">`
Because of this replacement all imported PCI interactions are broken.